src/parcel.vue

```diff
@@ -0,0 +1,24 @@
+<script>
+export default {
+  props: {
+    config: Object
+  },
+  methods: {
+    noConfig(createElement) {
+      return createElement(
```

Comment:
We should throw an Error when invalid props are passed, not render a div.

Reply:
- [ ] Throw error when there's `noConfig`